src/app/containers/EpisodeList/Episode.jsx

```diff
@@ -29,11 +29,15 @@ const TextWrapper = styled.div`
   }
 `;
 
-const Episode = ({ children, dir }) => {
+const Episode = forwardRef(({ children, dir }, ref) => {
   const showMediaIndicator = pathOr({}, '0', children).type !== Image;
 
   return (
-    <Wrapper dir={dir} showMediaIndicator={showMediaIndicator}>
+    <Wrapper
+      dir={dir}
+      showMediaIndicator={showMediaIndicator}
+      {...(ref && { ref })}
```

Comment:
does passing ref=undefined cause an issue?

Reply:
Nope, seems to work as expected 👍